content/en/docs/reference/access-authn-authz/certificate-signing-requests.md

```diff
@@ -107,7 +108,7 @@ Kubernetes provides built-in signers that each have a well-known `signerName`:
     1. CA bit allowed/disallowed - not allowed.
 
 1. `kubernetes.io/legacy-unknown`:  has no guarantees for trust at all. Some distributions may honor these as client
-  certs, but that behavior is not standard Kubernetes behavior.
+  certs, but that behavior is not standard Kubernetes behavior. Its not supported since api version `certificates.k8s.io/v1`.
```

Comment:
If there is an existing CSR that fetches via `/apis/certificates.k8s.io/v1beta1` with this legacy-unknown `signerName,` and then I try to fetch it as API version `/apis/certificates.k8s.io/v1`, what happens?

Reply:
you will get the `signerName: kubernetes.io/legacy-unknown`.
The v1 API simply blocks you from setting the `signerName` to `kubernetes.io/legacy-unknown` directly.

Reply:
@gaurav517: based on @tengqm's clarification, I recommend:
```suggestion
1. `kubernetes.io/legacy-unknown`: has no guarantees for trust at all. Some third-party
   distributions of Kubernetes may honor these as client certificates.  
   The stable CertificateSigningRequest API (version `certificates.k8s.io/v1`
   and later) does not let you set the `signerName` to `kubernetes.io/legacy-unknown`.
```

Reply:
👍  fixed in next commit.